### PR TITLE
chore: Add Hostname to Bugsnag Configuration

### PR DESCRIPTION
**Pull request by aarthykc**
## Context
This PR adds the hostname to the Bugsnag Configuration for better debugging. 

## Next Steps
- [x] Receive at least 1 recent approving review 
- [ ] Release a new version

### CODE REVIEW

File: nconf/bugsnag.go [hunk 1]
```diff
@@ -8,6 +8,7 @@ type BugSnagConfig struct {
 	Environment    string
 	APIKey         string `envconfig:"api_key" json:"api_key" yaml:"api_key"`
 	ProjectPackage string `envconfig:"project_package" json:"project_package" yaml:"project_package"`
+	HostNodeName   string `envconfig:"node_name" json:"node_name" yaml:"node_name"`
```
Comment (cbosss) [inline nconf/bugsnag.go:11]: Can you add a comment here... something like: 

`If left unset, bugsnag will default to the value returned by os.Hostname`.

Comment (mraerino) [inline nconf/bugsnag.go:11]: why not call the field just `NodeName`?

**Review by mraerino — COMMENTED** [review]
very nitty comment. code looks good.

File: nconf/bugsnag.go (hunk 1 above)
Comment (aarthykc) [reply nconf/bugsnag.go]: Donezo!

Comment (aarthykc) [reply nconf/bugsnag.go]: Donezo!

**Review by cbosss — APPROVED** [review]

**Pull request merged**